### PR TITLE
Explain limitation of multiple !include on top level

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Explain limitation of multiple !include on top level

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/splitting_configuration.markdown [hunk 1]
```diff
@@ -76,7 +76,10 @@ device_tracker: !include device_tracker.yaml
 
 Nesting `!include`s (having an `!include` within a file that is itself `!include`d) will also work.
 
-You can, as well, have multiple top-level `!include`s for a given integration, if you give a different label to each one:
+You can, as well, have multiple top-level `!include`s for platform integrations as well as the `automation`, `script` and `template` integrations, if you give a different label to each one. Configuration for other integrations can instead be split up by using packages. To
```
Comment (emontnemery) [inline source/_docs/configuration/splitting_configuration.markdown]: Do we explain what a "platform integration" is somewhere?

Comment (frenck) [reply source/_docs/configuration/splitting_configuration.markdown]: We don't explain that anywhere actually. It is also a legacy thing really, as they are no longer created and going away.

Comment (emontnemery) [reply source/_docs/configuration/splitting_configuration.markdown]: OK, we can reword to point to the `packages` feature as the preferred option and mention that the multiple-keys-with-shared-prefix works for certain integrations?

Comment (frenck) [reply source/_docs/configuration/splitting_configuration.markdown]: Yeah I think your latest adjustment is fine 👍

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery 👍

**Pull request merged**